kontrakter/dom/begjaeringDom/1.0/begjaeringDom.schema.json

```diff
@@ -0,0 +1,671 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://politiet.no/dom/1.0/begjaeringDom",
+  "description": "Begjæring om dom fra politiet til tingretten, ønsker dom avgjørelse tilbake.",
+
+  "type": "object",
+  "properties": {
+    "forsendelse": {
+      "$ref": "#/definitions/forsendelse",
+      "description": "avsender og mottaker"
+    },
+    "begjaeringDom": {
+      "$ref": "#/definitions/begjaeringDom"
+    },
+    "hovedStraffesak": {
+      "$ref": "#/definitions/straffesak",
+      "description": "Hoved straffesaken saksbehandler saken."
+    },
+    "dokumenter": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/dokument"
+      }
+    }
+  },
+  "required": ["forsendelse", "hovedStraffesak", "dokumenter"],
+  "additionalProperties": false,
+
+  "definitions": {
```

Comment:
Regner med flere skjema bruker noen av de samme definitions. Hvorfor er ikke det trukket ut i en egen fil så man slipper alle linjene med kode. Det gjør det tungt å sitt å se gjennom kontrakten

Reply:
Ryddet

Reply:
Vi har ikke delt opp i flere JSON Schema nå. Det må vi diskutere å bli enige om alle. 
* Se [Retningslinjer for modell...](https://domstol.atlassian.net/l/cp/a91qP4fF)
* [rfc/versjonsjhaandtering](https://github.com/domstolene/ESAS/blob/main/rfc/versjonshaandtering.md)
Jeg har laget noen maler som jeg kommer til å lage en egen PR på.
Gjenbruk at definisjoner er bra, men trenger at alle våre verktøy håndterer det eller vi kan lage en snutt som hjelper oss.